src/core/types/arbitrageloops/mempoolLoop.ts

```diff
@@ -198,6 +203,21 @@ export class MempoolLoop {
 			}
 		});
 	}
+
+	/**
+	 *
+	 */
+	public clearIgnoreAddresses() {
```

Comment:
cant you do this right when "keepIgnored" returns false? then you could remove it instantly no?